### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by teolemon**
Enjoy!

### CODE REVIEW

**Comment by stephanegigandet** [comment]
A few failing tests because of << site_name >>

**Comment by teolemon** [comment]
I'm going to fix the 200 files in Crowdin this time. That way we'll be at 0 weird diff permanently :-)

File: po/common/ku.po [hunk 1]
```diff
@@ -142,7 +142,7 @@ msgstr ""
 msgctxt "app_please_take_pictures"
 msgid "<p>This product is not yet in the <<site_name>> database. Could you please take some pictures of the product, barcode, ingredients list and nutrition facts to add it on <a href=\"https://world.openfoodfacts.org\" target=\"_blank\"><<site_name>></a>?</p>\n"
 "<p>Thanks in advance!</p>\n"
-msgstr "<p>Ev berhema e yet di ne <<site_name>> di nav Htmlê da ne. To lê zêde bike li ser products, barcodes, lîsteya maddeyên û maddeyên bi we hinek pictures <a href=\"https://world.openfoodfacts.org\" target=\"_blank\"><<site_name>></a>?</p>\n"
+msgstr "<p>Ev berhema e yet di ne<<site_nav>> di nav Htmlê da ne. To lê zêde bike li ser products, barcodes, lîsteya maddeyên û maddeyên bi we hinek pictures <a href=\"https://world.openfoodfacts.org\" target=\"_blank\"><<site_nav>></a>?</p>\n"
```
Comment (teolemon) [inline po/common/ku.po]: issue

File: po/openbeautyfacts/ko.po [hunk 2]
```diff
@@ -17,59 +17,59 @@ msgstr ""
...
 msgctxt "android_apk_app_link"
 msgid "https://world.openbeautyfacts.org/images/apps/obf.apk"
-msgstr ""
+msgstr "https://world.openbeautyfacts.org/images/apps/obf.apk"
 
 #. change hl=en by hl=fr or hl=de and make sure the url works
 msgctxt "android_app_link"
 msgid "https://play.google.com/store/apps/details?id=org.openbeautyfacts.scanner&hl=en"
-msgstr ""
+msgstr "https://play.google.com/store/apps/details?id=org.openbeautyfacts.scanner&hl=kr"
```
Comment (teolemon) [inline po/openbeautyfacts/ko.po]: ko

File: po/openbeautyfacts/ko.po [hunk 3]
```diff
@@ -17,59 +17,59 @@ msgstr ""
...
 msgctxt "ios_app_link"
 msgid "https://apps.apple.com/app/open-beauty-facts/id1122926380"
-msgstr ""
+msgstr "https://apps.apple.com/app/open-beauty-facts/id1122926380"
 
 #. make sure the image exists with your country code. Otherwise, ask @teolemon how to create a translated logo
 msgctxt "logo"
 msgid "openbeautyfacts-logo-en-178x150.png"
-msgstr ""
+msgstr "openfoodfacts-logo-kr-178x150.png"
```
Comment (teolemon) [inline po/openbeautyfacts/ko.po]: tofix

File: lang/fr/texts/nutriscore-formula.html [hunk 4]
```diff
@@ -1,258 +1,404 @@
...
+<ul>
+<li>Un nouveau champ "Fruits, légumes (minimum)" a été créé pour noter les valeurs des mentions "teneur minimum en fruits/légumes", etc.</li>
+<li>Par défaut, les produits de certaines catégories sont considérés comme ayant un contenu minimum de:
 <ul>
-<li>100% : <a href="http://fr.openfoodfacts.org/categorie/jus-de-fruits">jus de fruits</a></li>
-<li>90% : <a href="http://fr.openfoodfacts.org/categorie/compotes">compotes de fruits</a></li>
-<li>85% : <a href="http://fr.openfoodfacts.org/categorie/legumes">légumes</a></li>
-<li>50% : <a href="http://fr.openfoodfacts.org/categorie/confitures">confitures</a></li>
-<li>autres catégories à définir</li>
+<li>100%: <a href="https://world.openfoodfacts.org/category/fr:jus-de-fruits"> jus </a></li>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]:
```suggestion
<li>100%: <a href="https://world-fr.openfoodfacts.org/categorie/jus-de-fruits"> jus </a></li>
```

File: lang/fr/texts/nutriscore-formula.html [hunk 5]
```diff
@@ -1,258 +1,404 @@
...
+<li>90 % : <a href="https://world.openfoodfacts.org/category/fr:compotes">compote de fruits</a></li>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]:
```suggestion
<li>90 % : <a href="https://world-fr.openfoodfacts.org/categorie/compotes">compote de fruits</a></li>
```

File: lang/fr/texts/nutriscore-formula.html [hunk 6]
```diff
@@ -1,258 +1,404 @@
...
+<li>85 % : <a href="https://world.openfoodfacts.org/category/fr:legumes">légumes</a></li>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]:
```suggestion
<li>85 % : <a href="https://world-fr.openfoodfacts.org/categorie/legumes">légumes</a></li>
```

File: lang/fr/texts/nutriscore-formula.html [hunk 7]
```diff
@@ -1,258 +1,404 @@
...
+<li>50 % : <a href="https://world.openfoodfacts.org/category/fr:confitures">confitures</a></li>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]:
```suggestion
<li>50 % : <a href="https://world-fr.openfoodfacts.org/categorie/confitures">confitures</a></li>
```

File: lang/fr/texts/who-we-are.html [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
 
 <h2>Contributeurs</h2>
 
-<p>Les contributeurs au projet Open Food Facts sont des passionnés de nourriture. Ils ajoutent des produits à la base de données et ils sont aussi la source primaire pour les améliorations et pour les idées via
+<p>Les contributeurs au projet Open Food Facts sont des passionnés de nourriture. Ils ajoutent des produits à la base de données et ils sont aussi la source primaire pour les améliorations et pour les idées via
 le <a href="https://openfoodfacts.uservoice.com/">Forum d'idées</a> et plusieurs autres lieux de partage de discussion pour la communauté d'Open Food Facts.</p>
 
-<p>Open Food Facts est un projet ouvert à tous : vous pouvez nous rejoindre en <a href="https://fr.openfoodfacts.org/cgi/user.pl">créant un compte</a> en mois d'une minute, et vous pouvez ajouter votre premier produit en quelques minutes.</p>
+<p>Open Food Facts est un projet ouvert à tous : vous pouvez nous rejoindre en <a href="https://world-en.openfoodfacts.org/cgi/user.pl">créant un compte</a> en mois d'une minute, et vous pouvez ajouter votre premier produit en quelques minutes.</p>
```
Comment (teolemon) [inline lang/fr/texts/who-we-are.html]:
```suggestion
<p>Open Food Facts est un projet ouvert à tous : vous pouvez nous rejoindre en <a href="https://world-fr.openfoodfacts.org/cgi/user.pl">créant un compte</a> en mois d'une minute, et vous pouvez ajouter votre premier produit en quelques minutes.</p>
```

File: lang/fr/texts/nutriscore-formula.html [hunk 9]
```diff
@@ -1,258 +1,404 @@
...
 
-<p>La formule du score nutritionnel indiquées ci-dessus correspond à la formule du score nutritionnel de la Food Standards Agency (FSA) du Royaume Uni.</p>
+<p>La formule du score nutritionnel indiquée ci-dessus correspond à celle de la Food Standards Agency (agence des normes alimentaires) du Royaume-Uni.</p>
 
-<p>L'équipe du professeur Hercberg propose des adaptations pour le marché français qui sont mises en oeuvre
-pour le calcul du score sur Open Food Facts.</p>
+<p>L'équipe du professeur Hercberg a effectué des adaptations pour le marché français qui sont mises en œuvre pour calculer le score sur Open Food Facts.</p>
 
-<h4>Adaptation pour les fromages</h4>
+<h4>Adaptation pour le fromage</h4>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]: fail

File: lang/fr/texts/nutriscore-formula.html [hunk 10]
```diff
@@ -1,258 +1,404 @@
...
 
-<p>Pour les fromages, les points pour les protéines sont retranchés au score même si les points A sont ≥ 11.</p>
+<p>Pour les fromages, les points pour les protéines sont retranchés au score même si les points A sont supérieurs ou égaux à 11.</p>
 
-<h4>Adaptation pour les matières grasses</h4>
+<h4>Adaptation pour la graisse</h4>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]: fail

File: lang/pt_PT/texts/nutriscore.html [hunk 11]
```diff
@@ -134,43 +89,45 @@ <h2>Quais países adotaram o Nutri-Score?</h2>
...
 <h3>Petição para tornar o Nutri-Score obrigatório em todos os pacotes em toda a Europa</h3>
 
-<p>
-<a class="small button" href="http://www.pronutriscore.org"> Assine a petição Nutri-Score no site da comissão da UE </a> (requer uma forma válida de número de identificação europeu)
-</p>
+  <a class="small button" href="https://www.pronutriscore.org">Assine a petição Nutri-Score no site da comissão da UE</a> (requer uma forma válida de número de identificação europeu)
+
+  <p><a class="small button" href="https://twitter.com/share?url=https://world-en.openfoodfacts.org/nutriscore&amp;text=Join%20me%20in%20signing%20the%20%23PRONUTRISCORE%20petition&amp;hashtags=yeswescan">Tweet sobre a petição</a></p>
+  <p><a class="small button" href="https://www.facebook.com/sharer.php?u=https://world-en.openfoodfacts.org/nutriscore">Compartilhe esta chamada para fazer login no Facebook</a></p>
+  <p><a class="small button" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://world-en.openfoodfacts.org/nutriscore">Compartilhar esta chamada para fazer login no LinkedIn</a></p>
```
Comment (teolemon) [inline lang/pt_PT/texts/nutriscore.html]:
```suggestion
  <p><a class="small button" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://world-pt.openfoodfacts.org/nutriscore">Compartilhar esta chamada para fazer login no LinkedIn</a></p>
```

File: lang/pt_PT/texts/nutriscore.html [hunk 12]
```diff
@@ -134,43 +89,45 @@ <h2>Quais países adotaram o Nutri-Score?</h2>
...
+  </blockquote>
 <h3>Petição para tornar o Nutri-Score obrigatório em todos os pacotes em toda a Europa</h3>
 
-<p>
-<a class="small button" href="http://www.pronutriscore.org"> Assine a petição Nutri-Score no site da comissão da UE </a> (requer uma forma válida de número de identificação europeu)
-</p>
+  <a class="small button" href="https://www.pronutriscore.org">Assine a petição Nutri-Score no site da comissão da UE</a> (requer uma forma válida de número de identificação europeu)
+
+  <p><a class="small button" href="https://twitter.com/share?url=https://world-en.openfoodfacts.org/nutriscore&amp;text=Join%20me%20in%20signing%20the%20%23PRONUTRISCORE%20petition&amp;hashtags=yeswescan">Tweet sobre a petição</a></p>
+  <p><a class="small button" href="https://www.facebook.com/sharer.php?u=https://world-en.openfoodfacts.org/nutriscore">Compartilhe esta chamada para fazer login no Facebook</a></p>
```
Comment (teolemon) [inline lang/pt_PT/texts/nutriscore.html]:
```suggestion
  <p><a class="small button" href="https://www.facebook.com/sharer.php?u=https://world-pt.openfoodfacts.org/nutriscore">Compartilhe esta chamada para fazer login no Facebook</a></p>
```

File: lang/pt_PT/texts/nutriscore.html [hunk 13]
```diff
@@ -134,43 +89,45 @@ <h2>Quais países adotaram o Nutri-Score?</h2>
...
+  </ol>
+  </blockquote>
 <h3>Petição para tornar o Nutri-Score obrigatório em todos os pacotes em toda a Europa</h3>
 
-<p>
-<a class="small button" href="http://www.pronutriscore.org"> Assine a petição Nutri-Score no site da comissão da UE </a> (requer uma forma válida de número de identificação europeu)
-</p>
+  <a class="small button" href="https://www.pronutriscore.org">Assine a petição Nutri-Score no site da comissão da UE</a> (requer uma forma válida de número de identificação europeu)
+
+  <p><a class="small button" href="https://twitter.com/share?url=https://world-en.openfoodfacts.org/nutriscore&amp;text=Join%20me%20in%20signing%20the%20%23PRONUTRISCORE%20petition&amp;hashtags=yeswescan">Tweet sobre a petição</a></p>
```
Comment (teolemon) [inline lang/pt_PT/texts/nutriscore.html]:
```suggestion
  <p><a class="small button" href="https://twitter.com/share?url=https://world-pt.openfoodfacts.org/nutriscore&amp;text=Join%20me%20in%20signing%20the%20%23PRONUTRISCORE%20petition&amp;hashtags=yeswescan">Tweet sobre a petição</a></p>
```

File: lang/fr/texts/nutriscore-formula.html (hunk 9 above)
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]:
```suggestion
<h4>Adaptation pour les fromages</h4>
```

File: lang/fr/texts/nutriscore-formula.html (hunk 10 above)
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]:
```suggestion
<h4>Adaptation pour les matières grasses</h4>
```

File: lang/fr/texts/nutriscore-formula.html [hunk 14]
```diff
@@ -1,258 +1,404 @@
...
+  <td>&gt;80</td>
+  <td>&gt;3.5</td>
+  <td>&gt;8.0</td>
+</tr>
+</tbody>
 </table>
 
-<p>A noter : la pomme de terre, la patates douce, le taro, le manioc et le tapioca ne sont pas pris en compte dans les fruits,
-légumes et noix.</p>
+<p>Remarque : Les pommes de terre, le taro, le manioc et le tapioca ne sont pas considérés comme des fruits, légumes ou noix.</p>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore-formula.html]:
```suggestion
<p>Remarque : Les pommes de terre, les patates douces, le taro, le manioc et le tapioca ne sont pas considérés comme des fruits, légumes ou noix.</p>
```

File: lang/fr/texts/nutriscore.html [hunk 15]
```diff
@@ -79,109 +66,79 @@ <h3>Installer l'application mobile Open Food Facts</h3>
...
+  </ol>
+  </blockquote>
 <h3>Pétition pour rendre le Nutri-Score obligatoire sur tous les emballages de produits en Europe</h3>
 
-<p>
-<a class="small button" href="http://www.pronutriscore.org">Signez la pétition Nutri-Score sur le site de la Commission européenne</a> (nécessite un numéro d'identification européen valide)
-</p>
+  <a class="small button" href="https://www.pronutriscore.org">Signez la pétition Nutri-Score sur le site de la Commission européenne</a> (nécessite un numéro d'identification européen valide)
+
+  <p><a class="small button" href="https://twitter.com/share?url=https://world-en.openfoodfacts.org/nutriscore&amp;text=Join%20me%20in%20signing%20the%20%23PRONUTRISCORE%20petition&amp;hashtags=yeswescan">Tweeter sur la pétition</a></p>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore.html]:
```suggestion
  <p><a class="small button" href="https://twitter.com/share?url=https://world-fr.openfoodfacts.org/nutriscore&amp;text=Rejoignez%20moi%20et signez%20la%20petition%20%23PRONUTRISCORE&amp;hashtags=yeswescan">Tweeter sur la pétition</a></p>
```

File: lang/fr/texts/nutriscore.html [hunk 16]
```diff
@@ -79,109 +66,79 @@ <h3>Installer l'application mobile Open Food Facts</h3>
...
+  <p><a class="small button" href="https://www.facebook.com/sharer.php?u=https://world-en.openfoodfacts.org/nutriscore">Partager cet appel à signatures sur Facebook</a></p>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore.html]:
```suggestion
  <p><a class="small button" href="https://www.facebook.com/sharer.php?u=https://world-fr.openfoodfacts.org/nutriscore">Partager cet appel à signatures sur Facebook</a></p>
```

File: lang/fr/texts/nutriscore.html [hunk 17]
```diff
@@ -79,109 +66,79 @@ <h3>Installer l'application mobile Open Food Facts</h3>
...
+  <p><a class="small button" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://world-en.openfoodfacts.org/nutriscore">Partager cet appel à signatures sur LinkedIn</a></p>
```
Comment (teolemon) [inline lang/fr/texts/nutriscore.html]:
```suggestion
  <p><a class="small button" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://world-fr.openfoodfacts.org/nutriscore">Partager cet appel à signatures sur LinkedIn</a></p>
```

**Comment by teolemon** [comment]
ready for merging

**Pull request merged**